extras/preprocessors.py

```
@@ -82,3 +84,40 @@ def cpds(x):
 
 
 
+def simple_scribble_detector(ndarray_image, threshold=127):
```

Flake8 (7.2.0)

[error] too many blank lines (3)

(E303)